### DISCUSSION TITLE
What two continents are home to alligators? 🐊

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1198] [Galaxy Trivia] [answered]
**Bonus:** What is the last name of Barbie's friend Ken?

**Comment by peterbaccaro** [comment, marked as answer]
The two continents that are home to alligators are **Asia** and **North America**.

The last name of Barbie's friend Ken is **Carson**.

**Comment by Sajansharma0017** [comment]
China and the United States.
The last name of Barbie's friend Ken is Carson
